istio/install.go

```diff
@@ -37,18 +57,124 @@ func (istio *Istio) installIstio(del bool, version, namespace string) (string, e
 		return st, ErrMeshConfig(err)
 	}
 
-	err = istio.runIstioCtlCmd(version, del)
+	err = istio.applyHelmChart(del, version, namespace)
 	if err != nil {
-		istio.Log.Error(ErrInstallIstio(err))
-		return st, ErrInstallIstio(err)
+		return st, ErrApplyHelmChart(err)
 	}
 
+	// TODO: keep this as a fallback method
+	//err = istio.runIstioCtlCmd(version, del)
+	//if err != nil {
+	//	istio.Log.Error(ErrInstallIstio(err))
+	//	return st, ErrInstallIstio(err)
+	//}
+
 	if del {
 		return status.Removed, nil
 	}
 	return status.Installed, nil
 }
 
+func (istio *Istio) applyHelmChart(del bool, version, namespace string) error {
+	kClient := istio.MesheryKubeclient
+
+	// charts should be here ideally
+	repo := "https://gcsweb.istio.io/gcs/istio-release/releases/charts/"
```

Comment:
Is this the post-release processing location for helm templates (the artifact repositories for install bundles)?

Reply:
Yes, it is but not useful right now.
Ideally the index.yaml at this link should have metadata about all the charts but in this case its not updated with new releases.

Reply:
Just a placeholder for now.